examples/other.mdx

```diff
@@ -10,134 +10,19 @@ Once you are done, your project should be setup with all the required dependenci
 
 <CodeGroup>
 
-```rust Rocket
```

Comment:
without this "Rocket"/"Salvo"/"Poem"/etc here, the user is not going to make sense of these.

Two options:

1. We add sub-headings in `other.mdx` for each framework, and include the relevant files in the `CodeGroup` in each section.

2. We enhance the generator in `/import-examples` to add e.g. ```rust Rocket...``` at the start of each example that is being included in `other.mdx`